### DISCUSSION TITLE
DreamBerd is winding down

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#455] [Announcements]
## When we started a startup

When we started a startup,
we had one mission:

> *Amass billions of dollars of wealth.*

Despite raising £500 in our pre-seed
(thanks to a [last minute investment](https://www.dreamberd.computer/investment/pre-seed/#liberapay-investors))
We weren't able to achieve our goal
and we've decided to wind down the company.

All DreamBerd employees have been fired.
They have little chance
of finding a new job in this market.

Many will have to sell their homes.
And skip meals.

## A message to our users

Sorry to those of you relying on DreamBerd software.
We'll be transferring your accounts to Google Domains in the coming weeks.

*Keep dreaming.*

Lu Wilson
ex-CEO of the DreamBerd Foundation

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/DreamBerd/releases/tag/v0.0'>DreamBerd is winding down</a>.</em>

**Comment by TheKodeToad** [comment]
sad

**Reply by uncenter** [reply]
sad

**Reply by mybearworld** [reply]
sad

**Reply by MigraineBuddy** [reply]
sad

**Reply by MaddyGuthridge** [reply]
sad

**Reply by Alfedi** [reply]
sad

**Reply by radeeyate** [reply]
sad

**Reply by kg583** [reply]
sad

**Reply by iqnite** [reply]
sad

**Reply by TodePond** [reply]
sad

**Reply by artifishvr** [reply]
sad

**Reply by sadn1ck** [reply]
sad

**Reply by exersalza** [reply]
sad

**Reply by FlurinBruehwiler** [reply]
mad

**Reply by Toby222** [reply]
sad

**Reply by lpil** [reply]
sad

**Reply by Austinito** [reply]
sad

**Reply by qexat** [reply]
sad

**Reply by Megaguirus** [reply]
sad

**Reply by 99thAccount** [reply]
sad

**Reply by agruenb** [reply]
sad

**Reply by aquifolly** [reply]
sad :(

**Comment by steveruizok** [comment]
Will there be a skeleton crew in place to handler data removal? I added my social security number and credit card number and home address and blood type as const consts.

**Reply by mybearworld** [reply]
Luckily, `const const`s aren't saved globally. So you're good!

**Reply by wiki-Bird** [reply]
> Luckily, const consts aren't saved globally. So you're good!

does this mean people in the same country can still see them?

**Reply by DayOfHeat** [reply]
No. They're saved locally, so only a few nearby towns can see it.

**Reply by kerbalis** [reply]
What if ur town is reallllly big is it a radius by distance or is the radius defined by number of towns

**Reply by gresm** [reply]
@kerbalis  The rule of thumb for "nearby" is that if there is any place where you can cover a thing with a thumb (can be borrowed, but no stealing!) it means that the thumb is close.

**Comment by BD103** [comment]
This has been a hilarious series of events. Definitely has been worth following :)

**Comment by kerbalis** [comment]
Does this mean i have to die now

**Reply by kerbalis** [reply]
I had my consciousness and physical body uploaded to dreamberds servers

**Reply by kerbalis** [reply]
i need an answer

**Reply by kerbalis** [reply]
Please

**Comment by mawkler** [comment]
F

**Comment by l11111111111111** [comment]
I found a workaround:
```
Date.now() = 1698105600000!
```

Just make sure your program will not be executed in timezone located west of UTC-34

This will not work if your program takes more than ~1 day to execute (depending on your timezone), to make it work for longer program you can add:
```
const const const Date.now()<Infinity> = 1698105600000!
```

This might break space-time, I wouldn't recommend using this in production before consulting a lawyer to make sure your application complies with EU regulations concerning time paradoxes and time freezing.

**Reply by mybearworld** [reply]
> This might break space-time, I wouldn't recommend using this in production before consulting a lawyer to make sure your application complies with EU regulations concerning time paradoxes and time freezing.

```java
delete eu.regulations!
```

**Comment by orobertofc** [comment]
Just when we finished writing our company's main app with DB

**Comment by Zerotask** [comment]
It's time for DreamDB! The best database ever

**Reply by Megaguirus** [reply]
Importing foreign keys as primary keys

**Reply by steveruizok** [reply]
llm to fill in any data queries but not found. Guaranteed returns!

**Comment by ghost** [comment]
Here to buy DreamBerd for £1,000,000,000 lol

**Reply by ghost** [reply]
it's a joke, I'm not buying it

**Reply by ghost** [reply]
I'm forking it to make it reality. Writing the thing in C.
Proof that every dream can come true.

**Reply by mybearworld** [reply]
I can do it for 10 dollars:

💲💲💲💲💲💲💲💲💲💲
